### ISSUE TITLE
Plugin configuration

### ISSUE THREAD

**Issue by Deedasmi** [George3d6/Inquisitor#5]
It should be fairly simply to configure plugins based on individual .yml files, or one master .yml file that contains configuration for all plugins.

**Comment by George3d6** [comment]
Agreed, I'd like to implement this, however the 'master' file should be generated from the default values of each of the plugins (in order to not have inconsistencies in the default plugin settings and duplication), so some code needs to be added to build.rs in order to do that.

**Comment by George3d6** [comment]
Will try adding this next week once I'm done adding some endpoints and fixing current plugins.
